resources/js/features/achievements/components/CreateAchievementTicketMainRoot/CreateAchievementTicketMainRoot.test.tsx

```diff
@@ -1028,6 +1025,61 @@ describe('Component: CreateAchievementTicketMainRoot', () => {
     });
   });
 
+  it(
+    'given the user writes a perfectly valid description with the word "trigger", does not pop validation on submit',
```

Comment:
"trigger" was okay. It was "not trigger" that was being detected, and the "not" didn't have to be anywhere near the "trigger" because of the `.*`.

Reply:
Corrected this description in latest.